### PR TITLE
Polyhedron Demo: minor fixes and enhancements

### PR DESCRIPTION
**Pull request by maxGimeno**
## Summary of Changes
This PR
- fixes the behavior of the camera copy/paste function
- Reset the viewer's scene bbox when zooming to an item (from the context menu)
- Makes the transparency propagable to all selected items

### CODE REVIEW

**Comment by MaelRL** [comment]
Thanks!

In minor stuff that I've also just noticed, the "Select All Items" selects from first inserted to last inserted, but if you change the respective positions of the items in the list (with the arrows), then it does not select all the items anymore. For example, if you load `1-10.off`, close the 5th, re-open the 5th and move it to the 5th position in the list, if you Ctrl+A, you get items 1-5 and not 1-10.

**Comment by maxGimeno** [comment]
For the polylines there is already a PR fixing that.
I'll look into the ctrl+A stuff.

File: Polyhedron/demo/Polyhedron/MainWindow.cpp [hunk 1]
```diff
@@ -857,7 +857,7 @@ void MainWindow::updateViewerBBox(bool recenter = true)
   double l_dist = (std::max)((std::abs)(bbox_center.x - viewer->offset().x),
                       (std::max)((std::abs)(bbox_center.y - viewer->offset().y),
                           (std::abs)(bbox_center.z - viewer->offset().z)));
-  if((std::log2)(l_dist) > 13.0 )
+  if((std::log2)(l_dist) > 5.0 )
```
Comment (lrineau) [inline Polyhedron/demo/Polyhedron/MainWindow.cpp]: Why?

**Comment by MaelRL** [comment]
Adding to minor things, the visualization of element IDs could use some looking into, it has some really behavior when you zoom (just take any mesh, show IDs, and zoom with the scrollwheel).

**Comment by maxGimeno** [comment]
You mean when the camera is inside the mesh ?

**Comment by MaelRL** [comment]
Yes, or on the other side, facing away. Basically it seems that the IDs of all the elements are shown, even those behind the camera.

**Comment by maxGimeno** [comment]
I added a test to check the TextItem is not behind the nearPlane, it seems to work great.

File: Polyhedron/demo/Polyhedron/MainWindow.cpp [hunk 2]
```diff
@@ -1337,8 +1345,48 @@ void MainWindow::showSceneContextMenu(const QPoint& p) {
         QMenu menu;
         Q_FOREACH(QString name, menu_actions.keys())
         {
-          QAction* action = menu.addAction(name);
-          connect(action, &QAction::triggered, this, &MainWindow::propagate_action);
+          if(name == QString("alpha slider"))
+            continue;
+          if(name == QString("Alpha value"))
+          {
+            QWidgetAction* sliderAction = new QWidgetAction(&menu);
```
Comment (lrineau) [inline Polyhedron/demo/Polyhedron/MainWindow.cpp]: Missing `#include <QWidgetAction>` in `MainWindow.cpp`. Does it really compile on your machine? It fails on all test platforms:
https://cgal.geometryfactory.com/CGAL/testsuite/results-4.13-Ic-62.shtml#Polyhedron_Demo

Comment (maxGimeno) [reply Polyhedron/demo/Polyhedron/MainWindow.cpp]: Yes it really compiles on my machine, I wonder why. I guess my include tree in built differently or something.

**Pull request merged**